packages/sn-controls-react/src/viewcontrols/NewView.tsx

```diff
@@ -173,8 +160,4 @@ export class NewViewComponent<T extends GenericContent> extends Component<
   }
 }
 
-const newView = connect(
-  mapStateToProps,
-  mapDispatchToProps,
-)(NewViewComponent)
-export { newView as NewView }
+export { NewViewComponent as NewView }
```

Comment:
I think it would make sense to rename the component or remove this line. I don't see a value exporting this in 2 different name.